jobs/JRADAR_LEVEL2

```diff
@@ -83,36 +62,31 @@ export HOMEbufr=${HOMEbufr:-${OPSROOT}/../prod/decoders/decod_shared}
 export FIXbufr=$HOMEbufr/fix
 export DICbufr=$HOMEbufr/dictionaries
 
-#export HOMEsatingest=${HOMEsatingest:-$NWROOTp3/obsproc_satingest.${obsproc_satingest_ver}}
+#NO NEED for satingest now ushing tranjb module 
+## JY export HOMEsatingest=${HOMEsatingest:-$NWROOTp3/obsproc_satingest.${obsproc_satingest_ver}}
+#export HOMEsatingest=${HOMEsatingest:-${PACKAGEROOT}/obsproc_satingest.${obsproc_satingest_ver}}
 #export USHsatingest=${HOMEsatingest}/ush
 #export EXECobsproc_satingest=${HOMEsatingest}/exec
 
+## JY added the following - 10/20
+# export TRANush=${USHsatingest}/bufr_tranjb.sh
+
```

Comment:
Seems irrelevant to radarl2 processing.
Please remove.

Reply:
This is the comments from NCO. The above lines are still used on WCOSS. module of bufr_tranjb is replacing the directory paths on WCOSS2. However, NCO still want to know the difference between WCOSS and WCOSS2 during WCOSS2 transition so we can remove this in the next CDF when we fully transit to WCOSS2.

Reply:
That sounds fair enough.
I did not know the origins of these lines.